src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -607,16 +608,19 @@ public static unsafe class PowerShellUnsafeAssemblyLoad
         [UnmanagedCallersOnly]
         public static int LoadAssemblyFromNativeMemory(IntPtr data, int size)
         {
+            int result = 0;
             try
             {
                 using var stream = new UnmanagedMemoryStream((byte*)data, size);
                 AssemblyLoadContext.Default.LoadFromStream(stream);
-                return 0;
             }
             catch
             {
-                return -1;
+                result = -1;
             }
+
+            ApplicationInsightsTelemetry.SendUseTelemetry("PowerShellUnsafeAssemblyLoad", result == 0 ? "Success" : "Failure");
```

Comment:
Do we save on ingestion if we just send a 0 or 1?

Reply:
yes, i can make that change (it's still a string, though)